backend/dataall/core/permissions/api/resolvers.py

```diff
@@ -3,40 +3,29 @@
 
 from dataall.base.aws.sts import SessionHelper
 from dataall.base.aws.parameter_store import ParameterStoreManager
-from dataall.core.permissions.db.tenant_policy_repositories import TenantPolicy
+from dataall.base.db.exceptions import RequiredParameter
+from dataall.core.permissions.services.tenant_policy_service import TenantPolicyService
 
 log = logging.getLogger(__name__)
 
 
 def update_group_permissions(context, source, input=None):
-    with context.engine.scoped_session() as session:
-        return TenantPolicy.update_group_permissions(
-            session=session,
-            username=context.username,
-            groups=context.groups,
-            uri=input['groupUri'],
-            data=input,
-            check_perm=True,
-        )
+    if not input['groupUri']:
```

Comment:
It's rudimental check. Everything is validated inside `TenantPolicyService.update_group_permissions `, so it can be called not only from resolvers (if needed) safely.